python/paddle/fluid/framework.py

```diff
@@ -523,10 +524,15 @@ def find_name(var_list, name):
                             % (in_proto.name, len(in_args)))
                     in_arg_names = []
                     for arg in in_args:
-                        if isinstance(arg, basestring):
+                        if issubclass(arg.__class__, six.string_types):
                             in_arg_names.append(arg)
+                        elif isinstance(arg, six.binary_type):
+                            in_arg_names.append(arg.decode())
                         else:
-                            in_arg_names.append(arg.name)
+                            if issubclass(arg.name.__class__, six.string_types):
+                                in_arg_names.append(arg.name)
+                            elif isinstance(arg.name, six.binary_type):
+                                in_arg_names.append(arg.name.decode())
```

Comment:
else?

Reply:
`str` in Python2 was spilted into `str` and `bytes` in Python3, so here are two if condition